### ISSUE TITLE
What exactly is being finetunned using this repo?

### ISSUE THREAD

**Issue by fif911** [LennartPurucker/finetune_tabpfn_v2#9]
Hey, ProiorLabs team. First, great job on the framework and thanks for opensourcing it.

I want to better understand this fine-tuning repo. At first, I assumed that this fine-tuning would allow me to load the vanilla model and SFT (supervised fine-tuning) it on my data to add some domain knowledge. But what I saw from the interface it that the code mentions hyper parameter funetunning, which is different from actual funetunning, and the time limit confused me, as I want to be able to control how many backward passes are done.

Can you share what are the best practices for fine-tuning Tabpfn models and how to actually do SFT on TabPFN?

Thanks for your support.
Cheers, Oleksandr

**Comment by LennartPurucker** [comment]
Heyho!

This repo fine-tunes the weights of a TabPFN checkpoint. 
If you are talking about the examples, the code [there](https://github.com/LennartPurucker/finetune_tabpfn_v2/blob/master/examples/minimal_example.py#L22) talks about the hyperparameters OF finetuning (like the learning rate).

Each step is one backward pass. The code is set up to do as many backward passes as possible in the given time limit.
